code/create/dta_dicom2spec_rules.py

```diff
@@ -215,10 +160,36 @@ def _rules(self, series_dict, subject=None, anon_subject=None,
                 'bids-task': apply_bids_label_restrictions(_guess_task(self, series_dict) ),
                 'id': series_dict.get('SeriesNumber', 'unknown'),
 		'bids-run' : apply_bids_label_restrictions( run )
-              #  'bids-run' : apply_bids_label_restrictions(_guess_run(self, series_dict)) if "ProtocolName" in series_dict else None
                 }
 
+    # The table contains rows like "DTA003   T12122   T15722"
+    # so if you find a T-value in the table, you know, that the
+    # first field of that row contains the DTA003-value, that
+    # I have to use as the subject-ID
+    def get_column( self, t_value ):
+	# TODO: check if the relative path with .. in the beginning works.
+#         table_path = "/data/BnB1/DATA/source_data/DTA/code/dtage.tsv"
+         table_path = "../dtage.tsv"
+         with open( table_path, 'r' ) as table:
```

Comment:
With `..` you build a path relative to current working directory. So it depends on where this script is called from. But you are actually relying on a particular structure of the dataset, that you build. You can use `__file__` to refer to path of the very python file you are currently in. Build the path starting with that one rather than `..` and the reference is independent of where that code is called from.